specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -587,17 +590,17 @@ scope kafka
             string16 name;
         }
 
-        struct KafkaAlterConfigsResponseBeginEx
+        struct KafkaResponseAlterConfigsBeginEx
         {
             int32 throttle;
             KafkaResourceStatus[] resources;
         }
 
-        struct KafkaResponseBeginEx switch (uint8)
+        union KafkaResponseBeginEx switch (uint8)
         {
-            case 33 kafka::stream::KafkaAlterConfigsResponseBeginEx alterConfigs;
-            case 20: kafka::stream::KafkaDeleteTopicsResponseBeginEx deleteTopics;
-            case 19 kafka::stream::KafkaCreateTopicsResponseBeginEx createTopics;
+            case 33: kafka::stream::KafkaResponseAlterConfigsBeginEx alterConfigs;
+            case 20: kafka::stream::KafkaResponseDeleteTopicsBeginEx deleteTopics;
+            case 19: kafka::stream::KafkaResponseCreateTopicsBeginEx createTopics;
```

Comment:
```suggestion
            case 33: kafka::stream::KafkaAlterConfigsResponseBeginEx alterConfigs;
            case 20: kafka::stream::KafkaDeleteTopicsResponseBeginEx deleteTopics;
            case 19: kafka::stream::KafkaCreateTopicsResponseBeginEx createTopics;
```
I think this will read more naturally, and consistent with how we name for `KafkaBeginEx`, etc with `BeginEx` at the end.